tests/standalone/BotocoreUsing.py

```diff
@@ -0,0 +1,18 @@
+import botocore.session
+
+# nuitka-skip-unless-imports: botocore
+
+session = botocore.session.get_session()
+client = session.create_client('ec2', region_name='us-west-2')
+credentials = session.get_credentials()
+
+# these tests below require credentials (botocore.exceptions.NoCredentialsError: Unable to locate credentials)
```

Comment:
Is it possible to mock boto and perform API calls?

Reply:
Thanks, I'll get back on this some time next week. Very busy this weekend.

Reply:
Sure @tommyli3318. You can take a look at [moto](https://github.com/spulec/moto) and see if it's useful.

Reply:
It was very useful! I've updated it, please review.

Reply:
Since boto3 is built on botocore, do we need to test both of them or can we just skip botocore?